api/net/Areas/Editor/Controllers/StorageController.cs

```diff
@@ -196,5 +197,174 @@ public IActionResult Delete(string path, [FromRoute] string? location = "capture
         System.IO.File.Delete(safePath);
         return new JsonResult(info);
     }
+
+    /// <summary>
+    /// Make a clip from the target file identified in the clip parameter.
+    /// </summary>
+    /// <param name="fileName"></param>
+    /// <param name="directory"></param>
+    /// <param name="start"></param>
+    /// <param name="end"></param>
+    /// <param name="clipNbr"></param>
+    /// <returns></returns>
+    [HttpGet("clip")]
+    [Produces("application/json")]
+    [ProducesResponseType(typeof(FolderModel), (int)HttpStatusCode.OK)]
+    [ProducesResponseType((int)HttpStatusCode.BadRequest)]
+    [SwaggerOperation(Tags = new[] { "Storage" })]
+    public IActionResult CreateClip(string fileName, string directory, int start, int end, int clipNbr, string target)
+    {
+        var path = _config.CapturePath + directory;
+        var process = GetClipProcess(fileName, path, start, end, clipNbr, target);
+
+        process.Start();
+        process.WaitForExit();
+
+        var listing = new FolderModel(_config.CapturePath, directory);
+        return new JsonResult(listing);
+    }
+
+    /// <summary>
+    /// Make a clip from the target file identified in the clip parameter.
+    /// </summary>
+    /// <param name="fileName"></param>
+    /// <param name="directory"></param>
+    /// <returns></returns>
+    [HttpPut("join")]
+    [Produces("application/json")]
+    [ProducesResponseType(typeof(FolderModel), (int)HttpStatusCode.OK)]
+    [ProducesResponseType((int)HttpStatusCode.BadRequest)]
+    [SwaggerOperation(Tags = new[] { "Storage" })]
+    public IActionResult JoinClips(string fileName, string directory, string target)
```

Comment:
This probably should be an async function.

Reply:
Fixed.